data/sys_patch_dict.py

```diff
@@ -600,6 +600,10 @@ def _generate_sys_patch_dict(self):
                             "GeForceTeslaGLDriverWeb.bundle": "WebDriver-387.10.10.10.40.140",
                             "GeForceTeslaVADriverWeb.bundle": "WebDriver-387.10.10.10.40.140",
                         },
+                        "/System/Library/PrivateFrameworks": {
+                            # Restore OpenCL by adding missing compiler files
+                            "GPUCompiler.framework": "11.6" if os_data.os_conversion.is_os_newer(os_data.os_data.monterey, 0, self.os_major, self.os_minor) else {},
```

Comment:
iirc Python will interpret it as a conditional for "11.6" only:
```py
# If true:
"GPUCompiler.framework": "11.6",
# If false:
"GPUCompiler.framework": {},
```

This should wrap the entire entry as a conditional:
```py
**({ "GPUCompiler.framework": "11.6"} if os_data.os_conversion.is_os_newer(os_data.os_data.monterey, 0, self.os_major, self.os_minor) else {}),
```

Reply:
Oops, didn't copy the line correctly. Should be fixed in the new commit